workflow/workflow-demos-test/src_test/test/bpm/TestProcurementRequest.java

```diff
@@ -171,7 +171,7 @@ void approvalTask_verified_accepted(BpmClient bpmClient)
         .as().user("ldv")
         .execute();
     assertThat(result).isNotNull();
-    List<ITask> tasks = result.getRequestedCase().getActiveTasks();
+    List<ITask> tasks = result.workflow().nextTasks();
```

Comment:
It would be nice if we could select the nextTask by name, by part of the name, by activator, by activator name directly on result.workflow().
Currently we get the case or all next tasks and then select them by name or activator outside.
Can be done in an own PR of course.